volontulo/views/auth.py

```diff
@@ -24,6 +24,10 @@
 
 def login(request):
     u"""Login view."""
+    if '_auth_user_id' in request.session:
+        return redirect('/me')
+
+    user_form = UserForm()
```

Comment:
It's because related model class. Without this rendered form has no labels for email and password.